catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx

```diff
@@ -55,12 +56,16 @@ function createDescriptionList(
 
   return {
     title: item.title || '',
+    subheading: item.note || '',
     items: [
       { term: 'Location/shelfmark', description: shelfmark },
       accessMethod && { term: 'Access method', description: accessMethod },
       accessStatus && { term: 'Access status', description: accessStatus },
       accessTerms && { term: 'Access terms', description: accessTerms },
-      itemNote && { term: 'Item note', description: itemNote },
+      accessNote && {
+        term: 'Access note',
+        description: <WorkTitle title={accessNote} />, // TODO: Rename WorkTitle to e.g. ApiHtml
```

Comment:
I'm now not sure about using the `WorkTitle` (or any similar) component for this – I'm not sure what it adds, other than hiding the fact that we're using the `dangerouslySetInnerHtml` api. I'm tempted to update this to just be `<Fragment dangerouslySetInnerHtml={{__html: accessNote}} />`

Reply:
looks like you can't do that with `Fragment` and there's an RFC for a native React `RawHTML` element, but not landed. So I'll probably stick with a `span` for now.